DigitalLearningSolutions.Data.Tests/DataServices/GroupsDataServiceTests.cs

```diff
@@ -453,149 +453,130 @@ public void GetGroupCourse_returns_expected_course()
         [Test]
         public async Task DeleteGroupCustomisations_deletes_all_group_customisations()
         {
+            using var transaction = new TransactionScope(TransactionScopeAsyncFlowOption.Enabled);
+
             // Given
             const int groupId = 8;
 
-            using var transaction = new TransactionScope(TransactionScopeAsyncFlowOption.Enabled);
-            try
-            {
-                // When
-                groupsDataService.DeleteGroupCustomisations(groupId);
+            // When
+            groupsDataService.DeleteGroupCustomisations(groupId);
 
-                // Then
-                var groupCustomisations = await connection.GetGroupCustomisationIdsForGroup(groupId);
-                groupCustomisations.Should().BeEmpty();
-            }
-            finally
-            {
-                transaction.Dispose();
-            }
+            // Then
+            var groupCustomisations = await connection.GetGroupCustomisationIdsForGroup(groupId);
+            groupCustomisations.Should().BeEmpty();
+
+            transaction.Dispose();

```

Comment:
Not sure if you even need `transaction.Dispose()` given you've got `using` above.

Reply:
Yeah I don't think they are needed. I had left them in though. From a visual perspective I like to see the explicit dispose.
But maybe this is more about the transition from the older `using statements` where everything is "wrapped" over to the new `using declarations`.

Reply:
It's probably not going to get in the way in tests, so you're welcome to leave them in if you want. I don't see much benefit beyond being a reminder to readers that they should think about disposing objects when they're writing code, but feel free to try and convince me otherwise!